tests/unit_tests/test_wave_06.py

```diff
@@ -261,11 +253,37 @@ def test_swap_best_by_category_no_other_match_is_false():
         their_priority="Decor"
     )
 
-    raise Exception("Complete this test according to comments below.")
-    # *********************************************************************
-    # ****** Complete Assert Portion of this test **********
-    # *********************************************************************
-    # Assertions should check:
-    # - That result is falsy
-    # - That tai and jesse's inventories are the correct length
-    # - That all the correct items are in tai and jesse's inventories
+    assert not result
+    assert len(tai.inventory) == 3
+    assert len(jesse.inventory) == 3
+    assert item_a in tai.inventory
+    assert item_b in tai.inventory
+    assert item_c in tai.inventory
+    assert item_d in jesse.inventory
+    assert item_e in jesse.inventory
+    assert item_f in jesse.inventory
+
+def test_swap_by_newest():
+     # Arrange
+    item_a = Decor(condition=2.0, age=2)
+    item_b = Electronics(condition=4.0, age=5)
+    item_c = Decor(condition=4.0, age=3)
+    tai = Vendor(
+        inventory=[item_c, item_b, item_a]
+    )
+
+    item_d = Clothing(condition=2.0,age=7)
+    item_e = Decor(condition=4.0, age=1)
+    item_f = Clothing(condition=4.0, age=3)
+    jesse = Vendor(
+        inventory=[item_f, item_e, item_d]
+    )
+
+    # Act
+    result = tai.swap_by_newest(other_vendor=jesse)
```

Comment:
We could just pass in `jesse` as an argument to this method without using named parameters (especially since this method only takes one argument)
```suggestion
    result = tai.swap_by_newest(jesse)
```